api/utils.py

```diff
@@ -1,22 +1,15 @@
-# from https://github.com/hackoregon/transportation-system-backend/blob/staging/api/utils.py
-from django.test.runner import DiscoverRunner
+"""
+General utility functions used elsewhere.
+"""
 
 
-class UnManagedModelTestRunner(DiscoverRunner):
+def dictfetchall(cursor):
```

Comment:
I bet the original code in this file in the exemplar is needed for one of the boostrapping steps.  The `dictfetchall() function is definitely not needed.

Reply:
I believe this is part of @BrianHGrant's pytest setup.